core/src/main/java/org/kiwix/kiwixmobile/core/main/AssetUtils.kt

```diff
@@ -15,18 +15,18 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
+@file:JvmName("AssetUtils")
```

Comment:
So while this is nice for the time being I think we are fine not doing it. CoreReaderFragment only has so much time left existing in java seeing as Butterknife will stop working in AGP 5.0 (according to the latest build/lint warnings) and then it will just be a normal nice extension function.

Ideally I would like this as a private extension function in CoreReaderFragment because right now this has a lot of scope for such a minorly used function. Ideally Ideally there would be a separate class it could be a private extension in that encapsulates the entire feature of documentSections in CoreReader and it wouldn't be such a massive class but I'm rambling trying to come to a decision on best placement.

I think putting it in ContextExtensions is fine... then we can ponder its true place in a more kotlin-ified world